client/blocks/privacy-policy-banner/index.jsx

```diff
@@ -162,10 +162,10 @@ const mapStateToProps = state => {
 
 const mapDispatchToProps = {
 	acceptPrivacyPolicy: ( privacyPolicyId, privacyPolicyUserStatus ) =>
-		savePreference( PRIVACY_POLICY_PREFERENCE, {
-			...privacyPolicyUserStatus,
-			[ privacyPolicyId ]: true,
-		} ),
+		savePreference(
+			PRIVACY_POLICY_PREFERENCE,
+			Object.assign( {}, privacyPolicyUserStatus, { [ privacyPolicyId ]: true } )
```

Comment:
Why is `Object.assign` better than the spread operator here?

Reply:
`privacyPolicyUserStatus` can be `null`. For `Object.assign`, it's a valid parameter with a documented behavior: to ignore it.

I'm not so sure about the object spread operator. It's probably OK -- when testing it in Chrome console, `...null` works. But I couldn't find it documented on MDN, unlike `Object.assign`.

That's all, no big deal :)